presto-native-execution/presto_cpp/main/SessionProperties.cpp

```diff
@@ -497,6 +497,14 @@ SessionProperties::SessionProperties() {
       2147483647,
       QueryConfig::kQueryMemoryReclaimerPriority,
       std::to_string(c.queryMemoryReclaimerPriority()));
+
+  addSessionProperty(
+      kMaxNumSplitsListenedTo,
+      "Maximum number of splits to listen to per table scan node per worker.",
+      INTEGER(),
+      0,
+      QueryConfig::kMaxNumSplitsListenedTo,
```

Comment:
This query config is being added in  https://github.com/facebookincubator/velox/pull/13822.